### PR TITLE
Backports from develop to 2.6.5 release

### PR DESCRIPTION
**Pull request by droidmonkey**
[NOTE]: # ( Describe your changes in detail, why is this change required? )
[NOTE]: # ( Explain large or complex code modifications. )
[NOTE]: # ( If it fixes an open issue, please add "Fixes #XXX" )
* Auto-Type: Allow selection of modal dialogs on X11 [#6204]
* Persist Always On Top setting [#6236]
* Make KeePassXC start after the system tray is available on LXQt [#6216]
* Allow CSV import of bare TOTP secrets [#6211]
* OPVault: Use Text instead of Name for attribute and section names [#6334]
* Show countdown for clipboard clearing [#6333]
* Fix showing preview notes in an entry without notes [#6481]
* Fix unreachable setting of file permissions [#6514]
* Show search bar when toolbar is hidden or overflow [#6279]
* MinimizeAfterUnlock also when unlocking through browser [#6338]
* Set permissions of saved attachments to be private to the current user [#6363]
* Add command line option to lock open databases [#6511]
* Use application font size when setting default or monospace fonts [#6332]
* Allow resizing of reports table columns [#6435]
* Improve description text when changing Key File credential [#6396]
* Make Always on Top a local configuration setting
* Fix read usage in release-tool

## Testing strategy
[NOTE]: # ( Please describe in detail how you tested your changes. )
[TIP]:  # ( We expect new code to be covered by unit tests and documented with doc blocks! )
Tested on develop when merged and via unit tests.

### CODE REVIEW

**Comment by michaelk83** [comment]
Just a reminder re #6392. Seems like a fairly small yet useful fix to include in the backport?

**Comment by droidmonkey** [comment]
That needs to be a separate PR since the underlying code is entirely new.

**Comment by michaelk83** [comment]
I figured this *was* a separate PR from the original one. But alright, it's not that important, so I'll leave it at that.

**Comment by droidmonkey** [comment]
This is a backport pr which means no substantive edits from previously approved code.

File: src/core/Config.cpp [hunk 1]
```diff
@@ -92,6 +92,7 @@ static const QHash<Config::ConfigKey, ConfigDirective> configStrings = {
     {Config::GUI_HideToolbar, {QS("GUI/HideToolbar"), Roaming, false}},
     {Config::GUI_MovableToolbar, {QS("GUI/MovableToolbar"), Roaming, false}},
     {Config::GUI_HidePreviewPanel, {QS("GUI/HidePreviewPanel"), Roaming, false}},
+    {Config::GUI_AlwaysOnTop, {QS("GUI/GUI_AlwaysOnTop"), Roaming, false}},
```
Comment (phoerious) [inline src/core/Config.cpp:95]: IMHO this should be a local setting.

**Review by phoerious — APPROVED** [review]
I have two nitpicks, but they don't necessarily have to be fixed here in the backport. A fix on develop would be sufficient for me.

File: src/gui/Application.cpp [hunk 2]
```diff
@@ -317,13 +336,48 @@ bool Application::sendFileNamesToRunningInstance(const QStringList& fileNames)
     QByteArray data;
     QDataStream out(&data, QIODevice::WriteOnly);
     out.setVersion(QDataStream::Qt_5_0);
-    out << quint32(0) << fileNames;
+    out << quint32(0); // reserve space for block size
+    out << quint32(1); // ID for file name send. TODO: move to enum
+    out << fileNames; // send file names to be opened
+    out.device()->seek(0);
+    out << quint32(data.size() - sizeof(quint32)); // replace the previous constant 0 with block size
```
[the remark this answers is not in the document]
Comment (droidmonkey) [reply src/gui/Application.cpp:343]: That's a good idea, will smooth this out when I implement passing keyfile/yubikey to running instance.

File: src/core/Config.cpp (hunk 1 above)
Comment (droidmonkey) [reply src/core/Config.cpp:95]: Yes, yes it should be

**Pull request merged**